homeassistant/components/upnp/device.py

```diff
@@ -142,16 +142,28 @@ def model_name(self):
 
     async def async_get_total_bytes_received(self):
         """Get total bytes received."""
-        return await self._igd_device.async_get_total_bytes_received()
+        try:
+            return await self._igd_device.async_get_total_bytes_received()
+        except asyncio.TimeoutError:
+            _LOGGER.debug("Timeout during get_total_bytes_received")
```

Comment:
Shouldn't it be a warning or error log message?

Reply:
It used to be an error (implicit), which gave lots of tracebacks for some people due to broken devices.

I'll make it a warning.